### PR TITLE
fix #1381 test on travis out of memory

### LINKED ISSUE #1381: test on travis out of memory

### PR DESCRIPTION
**Pull request by huafengw**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -8,7 +8,7 @@ script:
 - echo "TRAVIS_TAG" $TRAVIS_TAG
 - echo "repo" $TRAVIS_REPO_SLUG
 - if [[ $TRAVIS_PULL_REQUEST != "false" || $TRAVIS_REPO_SLUG != "gearpump/gearpump" ]];
-    then sbt clean coverage +test;
+    then sbt -jvm-opts travis/jvmopts clean coverage +test;
```
Comment (stanleyxu2005) [inline .travis.yml]: As it's a concrete build environment profile, better move it to `project/travis/jvmopts`.

**Comment by huafengw** [comment]
The sbt test takes a very long time, maybe more than one hour. Hold off merge

**Comment by manuzhang** [comment]
the OOM seems not solved yet

**Comment by huafengw** [comment]
Passed

**Comment by manuzhang** [comment]
+1

**Pull request merged**